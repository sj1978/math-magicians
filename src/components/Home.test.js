/* eslint-disable linebreak-style */
import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

describe('Home', () => {
  test('renders Home component', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
