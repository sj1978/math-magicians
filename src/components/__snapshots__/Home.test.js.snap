// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders Home component 1`] = `
<section>
  <div
    className="home"
  >
    <h3
      className="home-title"
    >
      Welcome to our page!
    </h3>
    <article
      className="article"
    >
      Mathematics includes the study of such topics as numbers(artihmetic and number theory)includes(algebra)shapes and spaces in which they are contained (geometry) and quantities and their changes (calculus and analysis)here is no general consensus about its exact scope or epistemological status
    </article>
    <br />
    <article
      className="article"
    >
      Most of mathematical activity consists of discovering and proving (by pure reasoning) properties of abstract objects. These objects are either abstractions from nature (such as natural numbers or a line), or (in modern mathematics) abstract entities that are defined by their basic properties, called axioms. A proof consists of a succession of applications of some deductive rules to already known results, including previously proved theorems, axioms and (in case of abstraction from nature) some basic properties that are considered as true starting points of the theory under consideration. The result of a proof is called a theorem. Contrary to physical laws, the validity of a theorem (its truth) does not rely on any experimentation but on the correctness of its reasoning (though experimentation is often useful for discovering new theorems of interest).
    </article>
  </div>
</section>
`;
